frontend/helpers/context.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { ComponentProps, FC } from "react"
+import { AuthProvider } from "."
+import { FormProvider, useForm, useFormContext } from "react-hook-form"
+
+export const combineComponents = (...components: FC[]): FC => {
```

Comment:
Is this in use?

Reply:
Oh right. I was going to use it to combine Providers, only I couldn't get it to work. I couldn't get any global context decorators to work - I tried wrapping the FormProvider in the AuthProvider, or vice versa. They both worked within each story decorator, but I couldn't get anywhere with them in .storybook/preview.js.

Reply:
Update: I removed the context.tsx file, and instead changed providers.tsx so it wraps children in FormProvider as well as AuthProvider

Reply:
I think we need to put the global decorators in a separate export in preview.js rather than as part of parameters. Also, we should put the formProvider in preview rather than providers since the formprovider is not global in the real application, which also uses providers. I am able to provide auth and form contexts to all stories with this snippet (we would then remove the decorators from individual stories):

```javascript
// preview.js
export const decorators = [
  (Story) => (
    <Providers>
      <FormProvider {...useForm()}>
        <Story />
      </FormProvider>
    </Providers>
  )
]
```

Reply:
That looks great, and makes a lot more sense than having to write all that code into each story! (I just couldn't ever get global decorators to work.)